src/libcore/option.rs

```diff
@@ -880,7 +878,7 @@ impl<T> Option<T> {
     /// assert_eq!(old, None);
     /// ```
     #[inline]
-    #[unstable(feature = "option_replace", issue = "51998")]
+    #[stable(feature = "option_replace", since = "1.30.0")]
```

Comment:
1.30.0 seems unlikely? See https://forge.rust-lang.org/index.html for dates.

Reply:
It is unlikely to be merged before `Fri Oct 26 2018` ? You prefer `1.31.0` ?

Reply:
It has to go into beta first ;) i.e. 1.31.0 goes into beta on 2018-10-26.

Reply:
Ok I will change to `1.31.0` so but how do you know the `1.31.0` beta is set to this date ?

Reply:
Ho it starts when the previous version enter stable ? ok ! :)